tests/Oryx.Tests.Common/DockerVolume.cs

```diff
@@ -22,6 +22,15 @@ public class DockerVolume
 
         public const string ContainerDirRoot = "/oryxtests";
 
+        // TODO: rename Dir -> Path
+        public DockerVolume(
```

Comment:
Not making this public was deliberate since end users (us who write tests) do not hit into errors, so I am wondering if this change is really required for your use case.

Reply:
Well, I don't really need the whole mirror-copy thing that happens in `DockerVolume.Create`.. so the only other choice is introducing another static initializer (like renaming the current method to `CreateMirror` and then adding a no-op `DockerVolume.Create`)

Reply:
Yes, i was thinking the same thing...we should rather create another static member to abstract the user from this detail.